### ISSUE TITLE
UserWarning: Failed to create RectangleSelector object. Interactive pixel class labeling will be unavailable.

### ISSUE THREAD

**Issue by hong56123** [spectralpython/spectral#170]
Recently studied this library for processing hyperspectral data，This warning will appear when the function imshow（） is run, there is no problem with the graph, but you cannot select the region, where does this problem occur?
### code
```
import spectral
import cv2
img = spectral.open_image('F:/myPro/Pythonproj/MyQt/data/wheat-ref.hdr')
spectral.imshow(img,(96,59,21))
rgb = spectral.get_rgb(img,(96, 59, 21))
```
### Error message
```
D:\software\anaconda\envs\mycv\Lib\site-packages\spectral\graphics\spypylab.py:796: UserWarning: Failed to create RectangleSelector object. Interactive pixel class labeling will be unavailable.
  warnings.warn(msg)
```
### Technical details
Windows 10
x86_64
numpy: 2.1.1
spectral: 0.23.1
pillow: 10.2.0
wxPython: 4.2.2
matplotlib: 3.8.4
ipython: 8.20.0 
PyOpenGL: 3.1.7

**Comment by tboggs** [comment]
This is a duplicate of issue #154, which will be fixed shortly in version 0.24. In the meantime, you can access the fix earlier by installing from the develop branch of the repo.
